projects/plugins/jetpack/modules/carousel/jetpack-carousel.css

```
@@ -96,10 +98,16 @@ so we have to target all affected elements in touch devices.
 }
 
 .jp-carousel-overlay .swiper-container .swiper-button-prev {
-	left: 0px;
+	left: 0;
 }
+
 .jp-carousel-overlay .swiper-container .swiper-button-next {
-	right: 0px;
+	right: 0;
+}
+
+.jp-carousel-overlay .swiper-container.swiper-container-rtl .swiper-button-prev,
+.jp-carousel-overlay .swiper-container.swiper-container-rtl .swiper-button-next {
+    transform: scaleX( -1 );
```

You'll want to avoid using spaces here, so no one changes it to a tab later when editing that file.

```suggestion
	transform: scaleX( -1 );
```

---

Good spotting. 👍